api/server.js

```diff
@@ -95,6 +95,18 @@ app.use(function(req, res, next) {
 // TODO: investigate blocking writes to _users from the outside. Reads maybe as well, though may be harder
 //       https://github.com/medic/medic-webapp/issues/4089
 
+// When testing random stuff in-browser, it can be useful to access the database
+// from different domains (e.g. localhost:5988 vs localhost:8080).  Adding the
+// --allow-cors commandline switch will enable this from within a web browser.
+if(process.argv.slice(2).includes('--allow-cors')) {
```

Comment:
It is weird to me that we use environment variables for most stuff but are here using a command line flag. Consistency is nice

Reply:
Less environment variables?

Reply:
;¬)

Reply:
Also this is explicitly _only_ a dev option.  I don't think environment variables are a great way to control flags of this type.